edb/server/main.py

```diff
@@ -374,10 +375,21 @@ def server_options(func):
 
 
 def server_main(*, insecure=False, **kwargs):
-
     logsetup.setup_logging(kwargs['log_level'], kwargs['log_to'])
     exceptions.install_excepthook()
 
+    try:
+        fno_limits = resource.getrlimit(resource.RLIMIT_NOFILE)
+    except resource.error:
+        logger.warning('could not read RLIMIT_NOFILE')
+    else:
+        if fno_limits[0] < 2000:
```

Comment:
Can we define this as a constant, ideally with a comment.

Reply:
done